### ISSUE TITLE
New pipline in pulling state sometimes after creating/deleting the pipeline multiple times

### ISSUE THREAD

**Issue by SoheilSalehian** [pachyderm/pachyderm#575]
Hey!

I have this happening to me lately where I create a new pipeline run something. Then I get rid of the pipeline and the repo that the output is suppose to go to. Upon recreation, the job is stuck at "pulling":

```
ID                                 OUTPUT                                     STATE
4054afce32668693005fe490daec40ad   callocr/9835c9dd6ffa47c98218229e20613c5d   pulling
```

Here are the logs for the job for pipeline:

```
 | 2016-06-22T20:40:57Z ERROR fuse.DirectoryLookup {"directory":{"file":{"commit":{"repo":{"name":"nlp"},"id":"d7c2fc918e4343bfa14c85df498cde0e"}},"repoAlias":"out","write":true},"name":"nlp","err":"no such file or directory"}
```

Any ideas what could be causing this? I can fill in more details if you tell me where to look for them.

**Comment by SoheilSalehian** [comment]
looks like this issue was recently reported with #559, and a fix #574 is on its way! thanks guys.

**Comment by jdoliner** [comment]
Fix has been merged and is now present in master. Should be shipped as part of 1.1
